### ISSUE TITLE
[Assault] Mobs with icon next to their name in "Imperial Agent Rescue"

### ISSUE THREAD

**Issue by kaincenteno** [DerpyProjectGroup/topaz#1062]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: base

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

enemis have an "i" icon in the assault mission
![Screenshot from 2021-12-02 09-46-26](https://user-images.githubusercontent.com/26943220/144479809-b413a5e7-37d6-4bda-b41d-7c9188f44388.png)

**Comment by claywar** [comment]
namevis should most likely be 0 for these instance mobs.

**Comment by TeoTwawki** [comment]
this is caused by specific combinations of entityFlags combined with either namevis or. Changing either field should get rid of the icon. Note that I don't mean the _expected_ bit in entityFlags you would use to add the icon intentionally - there seems to be a combination of invalid things that will confuse the client that I have triggered myself before, and in my case toggling the bit at 0x00000040 got rid of it.


tl;dr Teo thinks we should just recapture these when possible.

**Comment by claywar** [comment]
This issue originally occurred with the introduction of AMAN Liaisons, and pretty much all mobs were impacted at the time (we were setting namevis to 1 for all of them); I'm fairly certain we mass changed to 0, but couldn't find the exact commit

**Comment by TeoTwawki** [comment]
> This issue originally occurred with the introduction of AMAN Liaisons, and pretty much all mobs were impacted at the time (we were setting namevis to 1 for all of them); I'm fairly certain we mass changed to 0, but couldn't find the exact commit

I am aware - turns out there are additional ways to make that `i` happen on mobs. If you want I can show you later.
